tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Controllers/AccountControllerTests.cs

```diff
@@ -261,5 +263,112 @@ public void Logout_WhenInvalidLogoutId_ShouldThrowArgumentNullException(string l
                 await sut.Logout(logoutId);
             });
         }
+
+        [Test]
+        public void ForgotPassword_ForgotPasswordViewModel_NullViewModel_ThrowsException()
+        {
+            static async Task ForgotPassword()
+            {
+                using var controller = new AccountControllerBuilder().Build();
+                await controller.ForgotPassword(null);
+            }
+
+            Assert.ThrowsAsync<ArgumentNullException>(ForgotPassword);
+        }
+
+        [Test]
+        public async Task ForgotPassword_ForgotPasswordViewModel_InvalidViewModel_ReturnsExpectedView()
+        {
+            using var controller = new AccountControllerBuilder()
+                .Build();
+
+            controller.ModelState.AddModelError(string.Empty, "Fake error!");
+
+            var expectedViewModel = new ForgotPasswordViewModel();
+            var result = await controller.ForgotPassword(expectedViewModel) as ViewResult;
+
+            Assert.NotNull(result);
```

Comment:
Use the FluentAssertion!

result.Should().NotBeNull()

Reply:
This is deliberate to prevent any static analysis tools from flagging `result` as may be null.

Reply:
Possibly another one to add to the next code standards meeting – do we care about the false positive 'may be null'? While I'm still not convinced by fluent assertions I know I'm outnumbered there so it may be that we stick with the fluent version regardless.

I'll leave it as is for now though.